patches/components-sync_bookmarks-bookmark_change_processor.cc.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/components/sync_bookmarks/bookmark_change_processor.cc b/components/sync_bookmarks/bookmark_change_processor.cc
+index 35a73a07bbcdd8aeba13acf37c6754aadb039768..ac7ebcbda6bc1de634b81be44ce98c2c3656b3ac 100644
+--- a/components/sync_bookmarks/bookmark_change_processor.cc
++++ b/components/sync_bookmarks/bookmark_change_processor.cc
+@@ -87,6 +87,7 @@ void BookmarkChangeProcessor::UpdateSyncNodeProperties(
+   bookmark_specifics.set_creation_time_us(src->date_added().ToInternalValue());
+   dst->SetBookmarkSpecifics(bookmark_specifics);
+   SetSyncNodeFavicon(src, model, dst);
++  brave_sync::AddBraveMetaInfo(src, model);
+   SetSyncNodeMetaInfo(src, dst);
+ }
+ 
+@@ -383,7 +384,7 @@ int64_t BookmarkChangeProcessor::UpdateSyncNode(
+ 
+ void BookmarkChangeProcessor::BookmarkMetaInfoChanged(
+     BookmarkModel* model, const BookmarkNode* node) {
+-  BookmarkNodeChanged(model, node);
++  // TODO(darkdh): make sure the meta info changed is what we want to sync
```

Comment:
?

Reply:
I should just remove the TODO. brave sync doesn't really sync meta info from bookmarks. The meta info we put in the sync protocol is for ease the friction for using chromium sync

Reply:
Actually we can support syncing meta info for bookmarks and move original meta info to records level which makes more senses because those fields are used by all sync types
https://github.com/brave/sync/pull/330

Reply:
Most importantly, this create an extra commit from chromium which is changed nothing for current brave_sync because we don't sync meta info.

Reply:
todo comment removed